src/tests/ftest/launch.py

```diff
@@ -1936,7 +1936,7 @@ def install_debuginfos():
 
     # Now install a few pkgs that debuginfo-install wouldn't
     cmd = ["sudo", "dnf", "-y"]
-    if is_el(distro_info):
+    if is_el(distro_info) or "suse" in distro_info.name.lower():
```

Comment:
There would be  case for a "is_suse()" function here.

Reply:
Perhaps, if we actually had any real SLES to test with.